corehq/warehouse/management/commands/commit_table.py

```diff
@@ -53,10 +44,10 @@ def handle(self, slug, **options):
         start = options.get('start')
         end = options.get('end')
 
-        model = SLUG_TO_STAGING_TABLE.get(slug)
+        model = get_cls_by_slug(slug)
         if not model:
             raise CommandError('{} is not a valid slug. \n\n {}'.format(slug, USAGE))
-        model.stage_records(start, end)
+        model.commit(start, end)
```

Comment:
yes that's the idea. for staging tables, `commit` is a `clear` then `load` command. for facts and dims, `commit` is just a `load` call